app/Resources/views/base.html.twig

```diff
@@ -7,11 +7,11 @@
         {% block stylesheets %}
         {# uncomment the following lines to compile SCSS assets with Assetic
 
-            {% stylesheets output="css/app.css"
-                "%kernel.root_dir%/Resources/assets/scss/bootstrap.scss"
-                "%kernel.root_dir%/Resources/assets/scss/font-awesome.scss"
-                "%kernel.root_dir%/Resources/assets/css/*.css"
-                "%kernel.root_dir%/Resources/assets/scss/main.scss"
+            {% stylesheets filter="scssphp" output="css/app.css"
+                "%kernel.root_dir%/../web/assets/scss/bootstrap.scss"
+                "%kernel.root_dir%/../web/assets/scss/font-awesome.scss"
+                "%kernel.root_dir%/../web/assets/css/*.css"
+                "%kernel.root_dir%/../web/assets/scss/main.scss"
```

Comment:
can be simpler: ``'assets/scss/main.scss``. Assetic will read relative paths from the web dir